LondonBritishLibrary/orient/BLorient596.xml

```diff
@@ -0,0 +1,223 @@
+  <title xml:lang="en" type="complete" ref="LIT2835RepCh119">To S. Michael</title>
+  <incipit xml:lang="gez">ሰላም፡ ለዝክረ፡ ስምከ፡ ምስለ፡ ስመ፡ ልዑል፡ ዘተሳተፈ፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.6">
+  <locus from="54v"/>
+  <title xml:lang="en" type="complete" ref="LIT2965RepCh246">To the angel Gabriel</title>
+  <incipit xml:lang="gez">በሰላመ፡ ገብሬል፡ መልአክ፡ በላዕለ፡ ማርያም፡ ዘአዕረፈ፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.7">
+  <locus from="60r"/>
+  <title xml:lang="en" type="complete">A prayer to our Lord</title>
+  <incipit xml:lang="gez">ኦእግዚእየ፡ ኢ'፡ ክ'፡ ወልደ፡ እግዚአብሔር፡ ሕያው፡ ወወልደ፡ ማርያም፡ ሥግው፡ ዕቀበኒ፡ እምኵሉ፡ እኩይ፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.8">
+  <locus from="71v"/>
+  <title>Another prayer</title>
+  <incipit xml:lang="gez">ኦእግዚእየ፡ ኢ'፡ ክ'፡ ወልደ፡ እግዚ'፡ ሕያው፡ በእንተ፡ ማርያም፡ ወላዲትከ፡ ርድአኒ፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.9">
+  <locus from="76v"/>
+  <title xml:lang="en" type="complete" ref="LIT2729RepCh13">A hymn</title>
+  <incipit xml:lang="gez">ሚካኤል፡ ዘትቀውም፡ በየማና፡ ለማርያም፡ ድንግል፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+</msContents>
+
+<physDesc>
+  <objectDesc form="Codex">
+      <supportDesc>
+          <support>
+              <material key="parchment"/>
+          </support>
+          <extent>
+              <measure unit="leaf">77</measure>
+              <measure unit="blank">3</measure>
+              <dimensions type="outer" unit="in">
+                  <height>6.125</height>
+                  <width>4.125</width>
+              </dimensions>
+              </extent>
+              <foliation>Two unnumbered blank leaves at the beginning and one at the end.</foliation>
+        </supportDesc>
+
+      <layoutDesc>
+          <layout columns="2" writtenLines="19"/>
+      </layoutDesc>
+  </objectDesc>
+
+  <handDesc>
+    <handNote xml:id="h1" script="Ethiopic">
+    <desc>Written in a good clear hand, of the <date notBefore="1800" notAfter="1900">19th Cent.</date></desc>
+    </handNote>
+    </handDesc>
+
+    <additions>
+    <list>
+
+<item xml:id="e1">
+<locus target="#22r #23v"/>
+<desc>Owners <persName ref="PRS13256GabraHanna">Gabra Ḥannā</persName> and <persName ref="PRS13253HaylaMikael">Ḫayla Mikāʾel</persName> mentioned on <locus target="#22r">f. 22r</locus>, <persName ref="PRS13254WalattaMaryam">Walatta Māryām</persName> on <locus target="#23v">f. 23v</locus>.</desc>
+</item>
+
+    </list>
+    </additions>
+
+    <bindingDesc>
+        <binding contemporary="true" xml:id="binding">
+            <decoNote xml:id="b1">In the original boards, lined with silk and covered with blind-tooled leather.</decoNote>
+            <decoNote xml:id="b2" type="bindingMaterial">
+                <material key="wood"/>
+                <material key="leather"/>
+                <material key="silk"/>
+            </decoNote>
+        </binding>
+    </bindingDesc>
+
```

Comment:
I think you could add information on the history, at least the the date of the origin can be estimated. Concerning the provenance, this one should also be from Medhane Alam.

Reply:
What is the reason to suspect that it is from Madhane 'Alam? Isn't the date it <handDesc> enough?

Reply:
I think that the manuscripts under these numbers are all from the Napier's expedition